tests/bcl-test/BCLTests/templates/iOSApp/ViewController.cs

```diff
@@ -56,7 +58,26 @@ protected virtual void TerminateWithSuccess ()
 		}
 #endif
 
-		public override void ViewDidLoad ()
+		public async Task<string[]> GetSkippedTests ()
+		{
+			var ignoredFiles = new List<string> ();
+			// the project generator added the required resources,
+			// we extract them, parse them and add the result
+			var executingAssembly = Assembly.GetExecutingAssembly();
+			foreach (var resourceName in executingAssembly.GetManifestResourceNames ()) {
```

Comment:
would not it be simpler to add files as "content" in the project ? and probe for `*.ignore` at runtime
that way a change to the ignore file won't trigger an assembly re-compilation (much slower for AOT)

Reply:
Maybe there could be more code sharing with XA if they already have this working ?

Reply:
Resolving since I moved it to content files.